model-optimizer/extensions/front/onnx/resize_ext.py

```diff
@@ -29,7 +30,22 @@ class ResizeExtractor(FrontExtractorOp):
     def extract(cls, node: Node):
         onnx_opset_version = get_onnx_opset_version(node)
         if onnx_opset_version is not None and onnx_opset_version >= 11:
-            raise Error("ONNX Resize operation from opset {} is not supported.".format(onnx_opset_version))
-        mode = onnx_attr(node, 'mode', 's', default=b'nearest').decode()
-        UpsampleOp.update_node_stat(node, {'mode': mode})
+            mode = onnx_attr(node, 'mode', 's', default=b'nearest').decode()
+            transformation_mode = onnx_attr(node,
+                                            'coordinate_transformation_mode',
+                                            's',
+                                            default=b'half_pixel').decode()
+            nearest_mode = onnx_attr(node, 'nearest_mode', 's', default=b'round_prefer_floor').decode()
+            extrapolation_value = onnx_attr(node, 'extrapolation_value ', 'f', default=0.0)
+            exclude_outside = onnx_attr(node, 'exclude_outside', 'f', default=0.0)
```

Comment:
This attribute is not handled in the MO code and it is not in the operation specification. Is it an error?

Reply:
Fixed.